### ISSUE TITLE
Too much namespacing

### ISSUE THREAD

**Issue by pumazi** [Connexions/cnx-epub#42]
After a publish the epub namespaces remain in the html content. We need to find a way to remove the namespaces on adaptation from epub to models.

Example paragraph tag after publish:

```html
<p xmlns="http://www.w3.org/1999/xhtml" xmlns:bib="http://bibtexml.sf.net/" xmlns:data="http://dev.w3.org/html5/spec/#custom" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:lrmi="http://lrmi.net/the-specification">test</p>
```

**Comment by amberwebb** [comment]
@pumazi  After adding processing instructions, publishing and then editing, the proc tags are littered with namespaces. Not very user friendly for the digital media specialists, not to mention it is a headache now to debug the html in the editor because of it.

**Comment by reedstrm** [comment]
the lxml XMLParser takes an ns_clean parameter: 
  >>> parser = etree.XMLParser(ns_clean=True)
  >>> tree   = etree.parse(StringIO(xml), parser)
  >>> etree.tostring(tree.getroot())
  b'<a xmlns="test"><b/></a>'

**Comment by pumazi** [comment]
This was temporarily fixed in Connexions/cnx-authoring#218. That fix needs removed and placed within cnx-epub.
